Mathlib/Geometry/Manifold/MFDeriv/SpecificFunctions.lean

```diff
@@ -660,6 +660,65 @@ theorem mfderiv_prod_eq_add_apply {f : M × M' → M''} {p : M × M'} {v : Tange
 
 end Prod
 
+section disjointUnion
+
+variable {M' : Type*} [TopologicalSpace M'] [ChartedSpace H M'] {s : Set (M ⊕ M')} {p : M ⊕ M'}
+
+/-- In extended charts at `p`, `Sum.swap` looks like the identity near `p`. -/
+lemma writtenInExtChartAt_sumSwap_eventuallyEq_id :
+    writtenInExtChartAt I I p Sum.swap =ᶠ[𝓝[range I] I (chartAt H p p)] id := by
```

Comment:
Tiny nitpick but the way this is written, it looks to me like `nhdsWithin` consumes three arguments rather than two. We should probably adjust some binding power in our notation but in the meantime could we write this in a way that makes it clear there are two args. Maybe:
```suggestion
    writtenInExtChartAt I I p Sum.swap =ᶠ[𝓝[range I] (I <| chartAt H p p)] id := by
```

Reply:
I tried using `<|` before and the binding power didn't work. Let me try your version...